src/styles/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.star {
```

Comment:
according to your naming, `star` and `stars` are separate blocks. Make `star` an element of `stars`